shell/browser/ui/inspectable_web_contents_view_mac.mm

```diff
@@ -60,4 +60,6 @@
   [view_ setTitle:base::SysUTF16ToNSString(title)];
 }
 
+const std::u16string& InspectableWebContentsViewMac::GetTitle() {}
+
```

Comment:
We need to return the title on macOS as well. As written this won't compile nor return the correct value on macOS.

`InspectableWebContentsViewMac::GetTitle()` should return something like `[view_ getTitle:base::SysNSStringToUTF16(title)]`

This also means you'll need to implement a `title` getter on `ElectronInspectableWebContentsView`.

Reply:
I have no experience developing on macOS, I implemented this function on macOS, but I don't know if it is correct and compiles because I don't have a macOS environment and can't verify it.